.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,44 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    # - name: make
+    #   run: make
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+    # required for buildx and multi platform build
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Log in to Docker Hub 
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.CR_USERNAME }}
+          password: ${{ secrets.CR_PAT }}
+      - uses: actions/checkout@v2
+      - name: Build container image
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          platforms: linux/amd64,linux/arm64
+          tags: |
+            ${{ secrets.CR_USERNAME }}/nettracer-bpf:latest
+            ${{ secrets.CR_USERNAME }}/nettracer-bpf:${{ github.run_number }}
```

Comment:
It'd be also useful to tag the image with NetTracer's version, e.g. 1.1.0 at the moment. Maybe it could even replace run number tag.

It can be read from [version.properties](https://github.com/dynatrace-oss/nettracer-bpf/blob/master/version.properties).